wasm/demo/src/main.js

```diff
@@ -72,3 +73,45 @@ snippets.addEventListener('change', updateSnippet);
 // Run once for demo (updateSnippet b/c the browser might try to keep the same
 // option selected for the `select`, but the textarea won't be updated)
 updateSnippet();
+
+const term = new Terminal();
+term.open(document.getElementById('terminal'));
+term.write(">>>> ");
```

Comment:
Rustpython uses 5 times `>`, since pypy already took the 4 times `>`. It would also be nice to retrieve this from `sys.ps1`. Try playing around with this in cpython / rustpython. You can modify your prompt.

Reply:
I changed the prompt to 5 `>`. As for taking it from `sys.ps1` I think there are a few more urgent stuff like backspace and multiple lines input first.